camelConnector/src/test/java/io/vantiq/extsrc/camelconn/connector/VantiqConfigurationTest.java

```diff
@@ -156,12 +155,13 @@ public void testRawNoRaw() {
     
         performConfigTest(name.getMethodName(), ROUTE_NOT_USED, "yaml",
                           null, props,
-                          (CamelContext runnerContext, Properties lclProps) -> {
-                                assert lclProps != null;
-                                assert propValues.size() == lclProps.size();
-                                propValues.forEach( (name, val) -> {
-                                    assert lclProps.containsKey(name);
-                                    assert lclProps.getProperty(name).equals(val);
```

Comment:
with setting props via initial properties, we can now fetch the values and have that work.  This is a more thorough test.